lib/bundle.js

```diff
@@ -14,6 +14,7 @@ var Location = require("./location");
 var bundleMrHtml = require("./bundle/mr");
 var bundleMontageHtml = require("./bundle/montage");
 
+var bundledModules = [];
```

Comment:
I think `bundledModules` need to be inside the closure of `bundle` function to make sure "module.exports = bundle;" do not leaks and can reused twice in a row.
You need to reset/or create bundledModules inside `bundle` function.